.github/workflows/main.yml

```diff
@@ -0,0 +1,46 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
```

Comment:
Please change this to trigger on any branch like this: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-using-a-list-of-events